catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -190,7 +196,15 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
     long totalHits = 0;
 
     try {
-      QueryResponse solrResponse = client.query(query, METHOD.POST);
+      QueryResponse solrResponse;
+
+      if (solrFilterDelegate.isIdQuery()) {
+        LOGGER.debug("Performing real time query");
+        SolrQuery realTimeQuery = getRealTimeQuery(query, solrFilterDelegate.getIds());
+        solrResponse = client.query(realTimeQuery);
```

Comment:
❗️ Query needs to be `METHOD.POST` to avoid URL length restrictions.

Reply:
I realized that and fixed it when I ran into the issue running the itests.